pkg/api/converterfromadmin.go

```diff
@@ -434,6 +434,14 @@ func mergeCertKeyPair(in *admin.Certificate, out *CertKeyPair) {
 	return
 }
 
+func mergeCertKeyChain(in *admin.CertificateChain, out *CertKeyPairChain) {
```

Comment:
mergeCertKeyPairChain

Reply:
can you add // TODO: this is not a great semantic - revisit if this field is ever enabled for write